### DISCUSSION TITLE
Option to force the same volume for all hitsounds globally regardless of beatmap

### DISCUSSION THREAD

**Discussion by SIUUUUUUUUUU17** [ppy/osu#30563] [Ideas]
Some maps makes using a single fixed hitsound really obnoxious because the volume is low in one part and high on another (moph's "Kakusei" chart being my worst offender), so having this would help

Project SEKAI doesn't have dynamic hitsound volumes and that's where my DIY edited Stepmania Ultralight skin's hitsounds come from

It can be a toggle that when enabled, the hitsounds will be controlled by the effects volume slider instead of the effects volume slider and the beatmap combined

**Comment by dddddjdddjddjdjdd** [comment]
this would be especially nice to have since a bunch of mania charts have the hitsound volume set ridiculously low (often 5%)

**Reply by bdach** [reply]
the mania people will probably say that's intentional, some people over there just don't want to bother with hitsounding at all

**Reply by dddddjdddjddjdjdd** [reply]
still, i wanna be able to hear my hitsounds without dropping the music to like 3%

**Comment by fleuph** [comment]
Please consider this. Many standard players play with loud hitsounds and rely on them for rhythm sense and timing. When elements that I hit don't play an audible hitsound, it makes me believe that I missed, and also takes away the rhythmic feedback from that hitobject. I understand the importance of hitsounding in mapping but since players are already given the option to ignore the actual hitsound files from a map can we also have the option to ignore hitsound volume?

**Comment by WalmartWhiteCat** [comment]
I really want this to be added so I can put hitsounds to 0% and still have those clean af menu sounds. The life of a hitsoundless player truly is jank.

**Reply by SIUUUUUUUUUU17** [reply]
This is not about a separate hitsound volume slider. This is about fixing the beatmap hitsound volume to the global effects volume slider

**Reply by WalmartWhiteCat** [reply]
Fuck.

**Reply by fleuph** [reply]
> I really want this to be added so I can put hitsounds to 0% and still have those clean af menu sounds.

Replace all of the [skinnable hitsound files](https://osu.ppy.sh/wiki/en/Skinning/Sounds#hitsounds) with empty audio files in whatever skin you prefer.

**Reply by N0Y0U2114** [reply]
Problem with that is for Lazer skins (argon and triangles) this will not work, as skins work differently, this is also seemingly how skins will work in the future (using global assets as opposed to skin specific) so that won't work going forward with the new skinning system. This is an important thing to get down early which is why it's being implemented here https://github.com/ppy/osu/pull/28733

**Comment by N0Y0U2114** [comment]
This kinda seems like a skin/mapper issues, although an overarching volume normalization system could be nice for when maps/skins are not well done

**Comment by kongehund** [comment]
If implementing, consider an additional option for whether this should also apply to 5% volume hitsounds, or alternatively, a different value to be forced for 5% volume hitsounds.

The reasoning behind this is that mappers will often use 5% volume to mute slider ends that don't represent any particular sound in the music. The player may wish to have these muted while still forcing regular hitsounds to the same volume level. 

For example, if a stream is mapped as 1/8 sliders instead of 1/4 circles, then the slider ends might be muted by having 5% volume. Having these be forced to the same level as the slider head hitsounds could make the section extremely "spammy", to the point that the player can't accurately use the hitsounds for rhythmic cueing.

**Comment by Newbie1st** [comment]
This is one of my favourite features in McOsu, mappers suddenly turning the volume of hitsounds to near zero to represent the song better can get extremely infuriating at times, as I don't ever use custom beatmap hitsounds and don't care about the song representation done that way. Please add this option.
